lib/charms/layer/basic.py

```diff
@@ -121,11 +122,20 @@ def init_config_states():
     from charms.reactive import set_state
     from charms.reactive import toggle_state
     config = hookenv.config()
+    config_defaults = {}
+    config_yaml = os.path.join(hookenv.charm_dir(), 'config.yaml')
+    if os.path.exists(config_yaml):
+        with open(config_yaml) as fp:
+            config_defs = yaml.load(fp).get('options', {})
+            config_defaults = {key: value.get('default', '')
```

Comment:
I carried that forward because I didn't realize we had `juju unset` already.  I'll fix.